target/linux/qualcommax/image/ipq50xx.mk

```
@@ -1,3 +1,25 @@
+DEVICE_VARS += BOOT_SCRIPT
+
+define Device/glinet_gl-b3000
+ $(call Device/FitImage)
+ $(call Device/UbiFit)
+  SOC := ipq5018
+  DEVICE_VENDOR := GL.iNET
+  DEVICE_MODEL := GL-B3000
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  NAND_SIZE := 128m
+  KERNEL_LOADADDR := 0x41080000
+  DEVICE_DTS_CONFIG := config@mp03.5-c1
+  SUPPORTED_DEVICES:=glinet,gl-b3000 b3000
```

Compatibility for `glinet,gl-b3000` is set by default, no need to add it.
```suggestion
  SUPPORTED_DEVICES += b3000
```

---

both are required, ipq-wifi will not load board bdf's without openwrt  style entry "glinet,gl-b3000". GL-Inet oem webui uses b3000 for firmware check and will fail without it.

---

luci also fails meta-data check with out the openwrt style entry..

---

did you (notice the) change from := to += ? because I haven't set this property and it works?

---

No i missed that.  I assume += is append and := is set ?  That would explain why it would fail using :=b3000 as i had tried in the past.  Theres my lesson for today :+1:  I will build and test.